oxo.py

```diff
@@ -1,26 +1,130 @@
 class OxoBoard:
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.number_to_win_for = 3
+        self.rows = 3
+        self.cols = 3
+
+        self.numberOfTiles = self.rows * self.cols
+
+        # Create and initialise the game board
+        self.board = []
+        for row in range(self.rows):
+            self.board += [[0] * self.cols]
+        print self.board
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+        return self.board[y][x]
```

Comment:
Is there a reason you chose to order the nested lists such that it's `[y][x]` rather than `[x][y]`? It doesn't really make a difference I suppose, but the latter might be more readable